src/ontology/cl-edit.owl

```diff
@@ -31919,6 +31920,17 @@ AnnotationAssertion(rdfs:label obo:CL_4040005 "mesenchymal stem cell of apical p
 SubClassOf(obo:CL_4040005 obo:CL_0000134)
 SubClassOf(obo:CL_4040005 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003677))
 
+# Class: obo:CL_4042001 (TAC3-positive medium spiny neuron)
```

Comment:
If you create two PRs one after another, you must manually increment the CL ID.

Reply:
thanks for letting me know! will do

Reply:
Hmmm - This should be fixed by ensuring Protege is set to remember the last added ID.  That way it increments itself.

Reply:
> Hmmm - This should be fixed by ensuring Protege is set to remember the last added ID. That way it increments itself.

I was not aware that this was a possibility across branches, how do you set it up? If I start a new branch, the new ID si the next one from the master, not from another unmerged branch I'm working on.

Reply:
Here's the checkbox to ensure you have checked. "Remember last ID between Protégé sessions"
<img width="1034" alt="Screenshot 2024-03-25 at 12 53 55" src="https://github.com/obophenotype/cell-ontology/assets/2208124/51a10d1d-f4ac-4b5a-8dd0-3ae8d8e103e2">

Reply:
Yes, I hae that selected, but it doesn't remember for different unmerged branches.

Reply:
It remembers only within the same branch I think